pipelines/tests/test_pipelines.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import os
+import sys
+
+from dwave.system.samplers import DWaveSampler
+
+try:
+    DWaveSampler()
+    _config_found = True
+except ValueError:
+    _config_found = False
+
+# /tests/integration/test_demo.py
+PY2 = sys.version_info.major == 2
+project_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+@unittest.skipUnless(_config_found, "no configuration found for accessing the D-Wave System")
```

Comment:
For the case of the Demos, once this PR is merged into the Master branch, this smoke test will have access to the API token and DWaveSampler. If it does not, I would want this smoke test to fail so that I would know that the Master branch is broken. Hence, I don't believe the `_config_found` is needed in this case.